src/modeler/AutoRest.Swagger/Validation/SummaryDescriptionValidation.cs

```diff
@@ -37,7 +37,7 @@ public class SummaryDescriptionValidation: TypedRule<Operation>
 
         public override IEnumerable<ValidationMessage> GetValidationMessages(Operation operation, RuleContext context)
         {
-            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.CurrentCultureIgnoreCase))
+            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.OrdinalIgnoreCase))
```

Comment:
I guessed so.. :-) You have just pointed me the one of the simplest pattern I hate most in C#..... :-)